app/templates/emails/invite_user_email.html

```diff
@@ -2,24 +2,22 @@
 <html>
 <head lang="en">
     <meta charset="UTF-8">
-    <title>Your Digital Marketplace invitation</title>
+    <title>Invitation to join {{ supplier }} as team member</title>
 </head>
 <body>
 	<p>Hello,</p>
 
-	<p>{{ user }} has invited you to become a contributor for {{ supplier }} on the Digital Marketplace.</p>
+	<p>{{ user }} has invited you to join {{ supplier }} as a team member on the Digital Marketplace.</p>
 
-	<p>Please <a href="{{ url }}">click this link</a> to accept the invitation</p>
+	<p>To accept, you must <a href="{{ url }}">register for a Digital Marketplace account</a> within 7 days.</p>
 
-	<p>You must click the above link within 7 days of receiving this email.</p>
-
-	<p>This is an automated message. Email <a href="mailto:marketplace@digital.gov.au">marketplace@digital.gov.au</a> if you need help.</p>
+	<p>If you have any questions or need guidance, please <a href="mailto:marketplace@digital.gov.au">contact the Marketplace</a>.</p>
```

Comment:
In the `create_user_error` template, you use `generic_contact_email` to pull the email address from config.  Should we make it consistent?

Reply:
updated